x/programs/runtime/runtime.go

```diff
@@ -24,11 +27,11 @@ const (
 	deallocFnName = "dealloc"
 )
 
-func New(log logging.Logger, meter Meter, storage Storage) *runtime {
+func New(log logging.Logger, meter Meter, db database.Database) *runtime {
```

Comment:
just a TODO for now but we don't want  db database.Database on New the runtime should be capable of using a vm defined storage.